src/components/FormElement/components/CodeElement/CodeElement.tsx

```diff
@@ -53,13 +53,13 @@ export const CodeElement: React.FC<Props> = ({ element, onChange }) => {
         language={element.language || CodeLanguage.JAVASCRIPT}
         showLineNumbers={true}
         showMiniMap={(element.value?.length || 0) > 100}
-        value={element.value || ''}
+        value={element.value?.replaceAll('\\n', '\n') || ''}
```

Comment:
This feels suspicious.... why is this only needed for `\n` and not other escapes (such as `\r`, or just `\` itself?)

Reply:
@asimonok Could you please comment?